### DISCUSSION TITLE
Absolute path to pino logger not included in the standalone bundle

### DISCUSSION THREAD

**Discussion by PedroJBrito** [vercel/next.js#44075] [Help]
Hi guys,

I've facing a situation that I really don't know how to solve. I'm using pino logger and I've created a custom transport that is passed to pino through an absolute path. However the bundle with next build works properly but the standalone version cannot find the module of the custom transport.

Any ideas about this issue or how to solve it?
Thanks a lot

**Comment by nemanjahifolks** [comment]
@PedroJBrito I have exact same issue in Vercel, how did you solve it?
